daprdocs/content/en/developing-applications/integrations/autoscale-keda.md

```diff
@@ -1,22 +1,27 @@
 ---
 type: docs
-title: "Autoscaling a Dapr app with KEDA"
-linkTitle: "Autoscale with KEDA"
+title: "How to: Autoscale a Dapr app with KEDA"
+linkTitle: "How to: Autoscale with KEDA"
 description: "How to configure your Dapr application to autoscale using KEDA"
-weight: 2000
+weight: 3000
 ---
 
 Dapr, with its modular building-block approach, along with the 10+ different [pub/sub components]({{< ref pubsub >}}), make it easy to write message processing applications. Since Dapr can run in many environments (e.g. VM, bare-metal, Cloud, or Edge) the autoscaling of Dapr applications is managed by the hosting layer.
```

Comment:
```suggestion
Dapr, with its building-block API approach, along with the many [pub/sub components]({{< ref pubsub >}}), makes it easy to write message processing applications. Since Dapr can run in many environments (for example VMs, bare-metal, Cloud or Edge Kubernetes) the autoscaling of Dapr applications is managed by the hosting layer.
```